docs/docs/cmd/onenote/notebook/notebook-add.md

```diff
@@ -0,0 +1,127 @@
+# onenote notebook add
+
+Create a new OneNote notebook.
+
+## Usage
+
+```sh
+m365 onenote notebook add [options]
+```
+
+## Options
+
+`-n, --name <name>`
+: Name of the notebook. Notebook names must be unique. The name cannot contain more than 128 characters or contain the following characters: ?\*/:<>
+
+`--userId [userId]`
+: Id of the user. Use either userId or userName, but not both.
+
+`--userName [userName]`
+: Name of the user. Use either userId or userName, but not both.
+
+`--groupId [groupId]`
+: Id of the SharePoint group. Use either groupName or groupId, but not both
+
+`--groupName [groupName]`
+: Name of the SharePoint group. Use either groupName or groupId, but not both.
+
+`-u, --webUrl [webUrl]`
+: URL of the SharePoint site.
+
+--8<-- "docs/cmd/\_global.md"
+
+## Examples
+
+Create a Microsoft OneNote notebook Private Notebook for the currently logged in user
+
+```sh
+m365 onenote notebook add --name "Private Notebook"
+```
+
+Create a Microsoft OneNote notebook Private Notebook in group 233e43d0-dc6a-482e-9b4e-0de7a7bce9b4
```

Comment:
we should not duplicate data from the example in the example description. So the description should be more general like in this case it could be 
```suggestion
Create a Microsoft OneNote notebook Private Notebook in a group by a given id
```

Reply:
the same should be corrected in other example descriptions below 👍